curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,527 @@ Which of the following is the default value of the `position` property?
 
 `static`
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which CSS property can be used to hide an element without removing it from the document flow?
+
+#### --distractors--
+
+`display: none`
+
+---
+
+`position: absolute`
+
+---
+
+`z-index: -1`
+
+#### --answer--
+
+`visibility: hidden`
+
+### --question--
+
+#### --text--
+
+Which CSS property allows you to offset an element from its normal position without removing it from the document flow?
+
+#### --distractors--
+
+`absolute`
+
+---
+
+`fixed`
+
+---
+
+`static`
+
+#### --answer--
+
+`relative`
+
+### --question--
+
+#### --text--
+
+What happens to an element with `position: absolute`?
+
+#### --distractors--
+
+It remains in the normal document flow.
+
+---
+
+It sticks to the viewport when scrolling.
+
+---
+
+It overlaps other elements without any control.
+
+#### --answer--
+
+It is removed from the normal document flow and positioned relative to its nearest positioned ancestor.
+
+### --question--
+
+#### --text--
+
+Which value of the `position` property allows an element to move relative to its normal position?
```

Comment:
this is your second question where the answer is relative. this question should be changed to something else.
My suggestion would be to pick something with a code example.
There are tons to choose from in the workshop here 

https://www.freecodecamp.org/learn/full-stack-developer/workshop-cat-painting/step-80